public/app/features/alerting/unified/Settings.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+import { LinkButton, Stack, Text } from '@grafana/ui';
+
+import { AlertingPageWrapper } from './components/AlertingPageWrapper';
+import { WithReturnButton } from './components/WithReturnButton';
+import { useEditConfigurationDrawer } from './components/settings/ConfigurationDrawer';
+import { ExternalAlertmanagers } from './components/settings/ExternalAlertmanagers';
+import InternalAlertmanager from './components/settings/InternalAlertmanager';
+import { SettingsProvider, useSettings } from './components/settings/SettingsContext';
+
+// @todo translate subtitle – move to navtree?
+const SUBTITLE =
+  'Manage Alertmanager configurations and configure where alert instances generated from Grafana managed alert rules are sent.';
+
+export default function SettingsPage() {
+  return (
+    <SettingsProvider>
+      <SettingsContent />
+    </SettingsProvider>
+  );
+}
+
+function SettingsContent() {
+  const [configurationDrawer, showConfiguration] = useEditConfigurationDrawer();
+  const { isLoading } = useSettings();
+
+  return (
+    <AlertingPageWrapper
+      navId="alerting-admin"
+      subTitle={SUBTITLE}
+      isLoading={isLoading}
+      actions={[
+        <WithReturnButton
+          key="add-alertmanager"
+          title="Alerting settings"
+          component={
+            <LinkButton href="/connections/datasources/alertmanager" icon="plus" variant="primary">
+              Add new Alertmanager
+            </LinkButton>
+          }
+        />,
+      ]}
+    >
+      <Stack direction="column" gap={2}>
+        {/* internal Alertmanager */}
```

Comment:
nit: we can keep the same naming we use in the UI (built-in/others) in the comment?